Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication.md

```diff
@@ -66,15 +67,19 @@ There is a specific documentation for load balancing with [Azure Web Apps](azure
 
 ## Examine Directory Factory Options
  
-- The `TempEnvDirectoryFactory` allows Examine to store indexes directly in the environment temporary storage directory, and should be used instead of `SyncTempEnvDirectoryFactory` mentioned above.
+- The `TempEnvDirectoryFactory` allows Examine to store indexes directly in the environment temporary storage directory, and should be used instead of `SyncTempEnvDirectoryFactory` mentioned below. The `TempEnvDirectoryFactory` should be used on front-end read-only servers.
+
+Add or update the 'Umbraco.Examine.LuceneDirectoryFactory' appSetting in your web.config as below:
 ```xml
 <add key="Umbraco.Examine.LuceneDirectoryFactory" value="Examine.LuceneEngine.Directories.TempEnvDirectoryFactory, Examine" />
 ``` 
-- The `SyncTempEnvDirectoryFactory` enables Examine to sync indexes between the remote file system and the local environment temporary storage directory, the indexes will be accessed from the temporary storage directory. This setting is needed because Lucene has issues when working from a remote file share so the files need to be read/accessed locally. Any time the index is updated, this setting will ensure that both the locally created indexes and the normal indexes are written to. This will ensure that when the app is restarted or the local environment temp files are cleared out that the index files can be restored from the centrally stored index files.
+- The `SyncTempEnvDirectoryFactory` enables Examine to sync indexes between the remote file system and the local environment temporary storage directory, the indexes will be accessed from the temporary storage directory. This setting is needed because Lucene has issues when working from a remote file share so the files need to be read/accessed locally. Any time the index is updated, this setting will ensure that both the locally created indexes and the normal indexes are written to. This will ensure that when the app is restarted or the local environment temp files are cleared out that the index files can be restored from the centrally stored index files. This is only required on the server serving the backoffice and not on front-end read-only servers.
```

Comment:
I think it's difficult to predict a particular system architecture and for this setting to be applied for front-end servers only. In the case of Azure WebApps, this is true, however, we specify that in the Azure specific docs https://github.com/umbraco/UmbracoDocs/blob/master/Getting-Started/Setup/Server-Setup/Load-Balancing/azure-web-apps.md